packages/console/src/console-history/ConsoleHistoryItem.scss

```
@@ -12,9 +12,31 @@ $button-vert-margin: ConsoleVariables.$button-vert-margin;
 }
 
 .console-history-item-command {
+  position: relative;
   white-space: pre-line;
 }
 
+.console-history-actions {
+  z-index: 1000;
+  position: absolute;
+  gap: $spacer-1;
+  top: 0;
+  right: $spacer-2;
+  display: none;
+  align-items: center;
+  background-color: var(--dh-color-neutral-hover-bg);
+  border-radius: $border-radius;
+  padding: $spacer-1;
+}
+
+.console-history-item-command-hovered {
```

You shouldn't need to do this yourself, you can just use the `:hover` pseudo-class:
```suggestion
.console-history-item-command:hover {
```

---

I originally had it with the pseudo-class, but the behavior is buggy. When I click on the contextual help button, the hover stops, and the contextual help popup moves to the top left corner of the screen (seemingly because the contextual help button is gone). Is there a good way to use the pseudo class that keeps it hovered after clicking on the contextual help?

---

Target the `:hover`, then when the tooltip is shown add another class that can also be targeted

---

done